lib/ripper-tags/vim_formatter.rb

```diff
@@ -59,6 +59,14 @@ def display_inheritance(tag)
       end
     end
 
+    def display_line_number(tag)
+      if options.line_number && tag[:line]
+        "\tline:%s" % tag[:line]
```

Comment:
Where is the `line:%d` syntax defined? Which types of tags processors support this field? I don't see this documented as part of "standard" tags file format: http://docs.ctags.io/en/latest/format.html

What `--excmd=number` does is that, instead of generating regex patterns in `display_pattern`, it outputs the line number in its place. If a function `foo` was defined on line 18, the resulting tags file would be:

```
foo{TAB}myfile.c{TAB}18;"{OTHER FIELDS}
```